modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/macros/APIExplorer.macro

```diff
@@ -139,6 +147,15 @@ definition {
 			value = "Execute Query (Ctrl-Enter)");
 	}
 
+	macro expandAPIMethod {
```

Comment:
Would you please update to expandAPIMethod() in executeAPIMethod() after adding this expandAPIMethod()?